services/azure-storage/src/provide_credential/azure_cli.rs

```diff
@@ -96,12 +96,12 @@ impl ProvideCredential for AzureCliCredentialProvider {
 
         // Calculate expiration time
         let expires_on = if let Some(timestamp) = token.expires_on_timestamp {
-            Some(chrono::DateTime::from_timestamp(timestamp, 0).unwrap())
+            Some(jiff::Timestamp::new(timestamp, 0).unwrap())
```

Comment:
Nit: `jiff::Timestamp::from_second`

Reply:
Yes. I keep the diff minor for review. Now we can use the tighter form :D